lib/ansible/modules/apt.py

```diff
@@ -82,6 +82,13 @@
          Please also see C(man apt-get) for more information.'
     type: bool
     default: 'no'
+  clean:
+    description:
+      - Run the equivalent of C(apt-get clean) to clear out the local repository of retrieved package files. It removes everything but
+        the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/.
+      - Recommended to run this as a stand-alone operation.
+    type: bool
+    default: 'no'
```

Comment:
missing version_added

Reply:
Would version_added be 2.13?

Reply:
yes

Reply:
Excellent, thank you for confirming. Added!